orga/app/styles/globals/pix-modal.scss

```diff
@@ -64,6 +64,23 @@
           letter-spacing: normal;
           font-size: 0.85rem;
         }
+
```

Comment:
Je propose pour le nom du commit
Add clipboard and tooltip css styles => Add style to clipboard

Reply:
Done